cypress_shared/pages/temporary-accommodation/manage/bedspaceShow.ts

```diff
@@ -80,4 +101,8 @@ export default class BedspaceShowPage extends Page {
         cy.get('a').contains('View').click()
       })
   }
+
+  clickLostBedLink(premisesId: string, roomId: string, lostBedId: string): void {
+    cy.get(`[href="${paths.lostBeds.show({ premisesId, roomId, lostBedId })}"]`).click()
```

Comment:
Consider changing so it's navigating by the link text? I feel like part of the test is "does the text as the user sees it go to the right place"

Reply:
Ah, there might not be a nice way of doing that, right, since you don't have a CRN like with bookings?

Reply:
yeah, exactly